Source/WebCore/html/HTMLFormControlElement.cpp

```diff
@@ -423,69 +423,69 @@ RefPtr<Element> HTMLFormControlElement::invokeTargetElement() const
     if (!canInvoke(*this))
         return nullptr;
 
-    return getElementAttribute(invoketargetAttr);
+    return getElementAttribute(commandforAttr);
 }
 
 constexpr ASCIILiteral togglePopoverLiteral = "togglepopover"_s;
 constexpr ASCIILiteral showPopoverLiteral = "showpopover"_s;
 constexpr ASCIILiteral hidePopoverLiteral = "hidepopover"_s;
 constexpr ASCIILiteral showModalLiteral = "showmodal"_s;
 constexpr ASCIILiteral closeLiteral = "close"_s;
-InvokeAction HTMLFormControlElement::invokeAction() const
+CommandType HTMLFormControlElement::commandType() const
 {
-    auto action = attributeWithoutSynchronization(HTMLNames::invokeactionAttr);
+    auto action = attributeWithoutSynchronization(HTMLNames::commandAttr);
```

Comment:
This should probably be named `command`? Although the specification PR still calls some things action?